src/main/kotlin/br/com/zup/op/events/exception/model/ApiFieldError.kt

```diff
@@ -0,0 +1,8 @@
+package br.com.zup.op.events.exception.model
```

Comment:
remove this package `exception.custom` and move this class to `infra.validation` package